design/admin/templates/ajaxuploader/location.tpl

```diff
@@ -3,6 +3,20 @@
     <fieldset>
         <legend>{"Step 2/3: Choose a location for the new '%class' object"|i18n( 'design/admin/ajaxuploader', '', hash( '%class', $class.name|wash() ) )}</legend>
         <p>{'<em>%file</em> has successfully been uploaded.'|i18n( 'design/admin/ajaxupload', '', hash( '%file', $file.original_filename|wash() ) )}</p>
+        {if $class.identifier|eq( 'image' )}
```

Comment:
I should check it against the setting then I think.  i.e. find MimeClassMap[image] and check against that value